src/args.ts

```diff
@@ -7,6 +7,7 @@ const args = nomnom.script('ecmarkup')
     help: { abbr: 'h', flag: true, help: 'Display this help message' },
     watch: { abbr: 'w', flag: true, help: 'Rebuild when files change' },
     biblio: { abbr: 'b', metavar: 'FILE', help: 'Write a biblio file to FILE' },
+    omitECMA262Biblio: { full: 'omit-ecma262biblio', flag: true, default: false, help: 'Skip loading ECMA-262 biblio file' },

```

Comment:
Couldn't it be

```js
ecma262biblio: { full: 'ecma-262-biblio', flag: true, default: true, help: 'Load ECMA-262 biblio file' },
```

and then from the CLI we do `--no-ecma-262-biblio`?

Reply:
Ah, that does seem better indeed. Will do!

Reply:
Done!